documentation/docs/Contributing/Contributing_to_repository/development-pipeline.md

```diff
@@ -18,7 +18,7 @@ If you are ever unsure about one of the steps described in this document, please
 
 ## Step One: Get the latest version of BrainFANS
```

Comment:
Steps 1 & 2 could happen either way around. I have a preference for Issue first as it make this page a more universal guide for how to make a change, whether you came up with the improvement yourself or you picked it up from the issues?

Reply:
That's a good point.